lib/galaxy/dependencies/dev-requirements.txt

```diff
@@ -145,10 +145,10 @@ pydantic==1.7.3; python_version >= "3.6" and python_version < "4.0"
 pydot==1.4.2; python_version >= "3.6" and python_full_version < "3.0.0" and python_version < "4" or python_version >= "3.6" and python_version < "4" and python_full_version >= "3.4.0"
 pyeventsystem==0.1.0
 pyfaidx==0.5.9.5
-pygithub==1.54.1; python_version >= "3.6"
+pygithub==1.53; python_version >= "3.5"
```

Comment:
this here is a downgrade

Reply:
Yes, we need to look into it. Last time we had a troubling downgrade of fastapi that happened like this: https://github.com/galaxyproject/galaxy/pull/11628#discussion_r595680035 . That one was easily fixable by simply pinning the dependency. We need to check why this one happened (and it might be not critical). I don't want to touch it until Monday though..

Reply:
I'm investigating this.

Reply:
Cause of issue: pygithub (1.54.1) depends on pyjwt (<2.0)
Poetry upgrades pyjwt from 1.7.1 to 2.0.1. This causes subsequent downgrade of pygithub to 1.54, but 1.54 is incompatible with requests (>=2.14.0,<2.25). So Poetry downgrades it to 1.53, which does not have the <2.25 constraint.
I've pinned pygithub to >=1.54.1 - I think this should solve it, as the ONLY other package that depends on pyjwt is social-auth-core, and it depends on >=1.7.1.